sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorBuilder.java

```diff
@@ -60,6 +66,12 @@ long getScheduleDelayNanos() {
     return scheduleDelayNanos;
   }
 
+  // Visible for testing
+  public BatchSpanProcessorBuilder setQueue(Queue<ReadableSpan> queue) {
```

Comment:
We don't need a test for normal queue with batchspanprocessor since it's not really supported. Let's revert those bits and just add a test in JcTools for the static method itself

Reply:
@anuraaga Added a new test class for JcTools covering all cases. 

<img width="671" alt="Screen Shot 2022-07-03 at 12 17 03 AM" src="https://user-images.githubusercontent.com/37902584/177029402-b73d39aa-e528-4120-90ef-eac5b834df8d.png">